pkg/operator/ceph/file/controller.go

```diff
@@ -369,8 +375,10 @@ func (r *ReconcileCephFilesystem) reconcile(request reconcile.Request) (reconcil
 
 				// update ObservedGeneration in status at the end of reconcile
 				// Set Ready status, we are done reconciling
-				r.updateStatus(observedGeneration, request.NamespacedName, cephv1.ConditionReady, opcontroller.GenerateStatusInfo(cephFilesystem))
-				statusUpdated = true
+				if r.updateStatus(observedGeneration, request.NamespacedName,
```

Comment:
nit: Args are best either all on the same line or all on different lines, IMO, so i'd suggest keeping the single line

Reply:
Done